web/docs/language/features.md

```diff
@@ -443,6 +443,149 @@ import { isPrismaError, prismaErrorToHttpError } from '@wasp/utils.js'
+
+### Job Executors
+
+Job executors handle the scheduling, monitoring, and execution of our job functions. They are the primary abstraction we provide that allows you to write one `async` JavaScript `job` handler function, but have it executed in many different job queue contexts by changing a single line in your `.wasp` file.
+
+#### pg-boss
+
+We have selected [pg-boss](https://github.com/timgit/pg-boss/) as our first job executor to handle the low-volume, basic job queue workloads many web applications have (think sending emails, connecting to external HTTP APIs, etc.). By using PostgreSQL (and [SKIP LOCKED](https://www.2ndquadrant.com/en/blog/what-is-select-skip-locked-for-in-postgresql-9-5/)) as it's storage and synchronization mechanism, it allows us to provide many job queue pros without any additional infrastructure or complex management.
+
+### Basic job definition and usage
+
+To register a `job` in Wasp, simply add a declaration with a reference to an `async` function, like the following:
+
+```css title="main.wasp"
+job mySpecialJob {
+  executor: PgBoss,
+  perform: {
+    fn: import { foo } from "@ext/jobs/bar.js",
+    options: {=json { "retryLimit": 1 } json=} // optional
+  }
+}
+```
+
+Then, in your [Operations](/docs/language/features#queries-and-actions-aka-operations) or [setupFn](/docs/language/features#setupfn-extimport-optional), you can enqueue work to be done:
+```js
+import { mySpecialJob } from '@wasp/jobs/mySpecialJob.js'
+
+const submittedJob = await mySpecialJob.submit({ job: "args" })
+console.log(await submittedJob.pgBoss.details())
+
+// Or, if you'd prefer it to execute in the future, just add a .delay().
+// It takes a number of seconds, Date, or ISO date string.
+await mySpecialJob.delay(10).submit({ job: "args" })
+```
+
+And that is it! Your job will be executed by the job executor as if you called `foo({ job: "args" })`.
+
+### Recurring jobs
+
+If you have work that needs to be done on some recurring basis, you can add a `schedule` to your job declaration:
+
+```css  {6-10} title="main.wasp"
+job mySpecialJob {
+  executor: PgBoss,
+  perform: {
+    fn: import { foo } from "@ext/jobs/bar.js"
+  },
+  schedule: {
+    cron: "0 * * * *",
+    performFnArg: {=json { "job": "args" } json=}, // optional
+    options: {=json { "retryLimit": 2 } json=} // optional
+  }
+}
+```
+
+In this example, you do _not_ need to invoke anything in JavaScript. You can imagine `foo({ "job": "args" })` getting automatically scheduled and invoked for you every hour.
+
+### Fields
+
+#### `executor: JobExecutor` (required)
+The executor can be either `Passthrough` or `PgBoss`. `Passthrough` is a zero-dependency executor used mainly for testing purposes. `PgBoss` is the recommended executor for low-volume, production use cases, and requires your `app.db.system` to be `PostgreSQL`.
```

Comment:
If we are mentioning `Passthrough`, then I think we should explain it the same way as we are explaining the `PgBoss`, with a special section. I am wondering if we could give it a nicer name, smth like `Simple`, or `Basic`, or even `Dummy`.
Do we even want users to be able to use such executor, what do you think? If we don't then we don't need to document it, but if we do, we should.

Reply:
Yeah that name leaves something to be desired. I am not sure they even need to know about it. We could change the name to `Basic` internally. But as I think about it, I'm not entirely sure we even need it 🤔 it was helpful to abstract the patterns for 2 or more types of Jobs, but in reality, I am unsure if we will ever use it ourselves. Maybe I will just change the term to `Simple` and we not document, and then possibly remove entirely in future on next Job addition.